.github/workflows/wheels.yml

```diff
@@ -143,10 +150,20 @@ jobs:
         uses: pypa/cibuildwheel@v2.19.2
         with:
          package-dir: ./dist/${{ startsWith(matrix.buildplat[1], 'macosx') && env.sdist_name || needs.build_sdist.outputs.sdist_file }}
+        if: ${{ matrix.buildplat[1] != 'pyodide_wasm32' }}
         env:
           CIBW_PRERELEASE_PYTHONS: True
           CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}
 
+      - name: Build wheels (Pyodide)
+        uses: pypa/cibuildwheel@v2.19.2
+        with:
+          package-dir: ./dist/${{ startsWith(matrix.buildplat[1], 'macosx') && env.sdist_name || needs.build_sdist.outputs.sdist_file }}
+        if: ${{ matrix.buildplat[1] == 'pyodide_wasm32' }}
+        env:
+          CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}
+          CIBW_PLATFORM: pyodide

```

Comment:
You can combine this step with the others by doing

`CIBW_PLATFORM: ${{ matrix.buildplat[1] == 'pyodide_wasm32' && 'pyodide' || 'auto }}`

in the step above, I think

Reply:
Thanks, @lithomas1 – let me try

Reply:
Addressed via 8779d8717a55325675806130e9122cc82276c7ac and 5106edd8800b8b43028980cd4cb49d2d3ce1ae89, the job is running successfully at https://github.com/pandas-dev/pandas/actions/runs/9912096788/job/27386251301